test/mocks/MockAppConfig.scala

```diff
@@ -27,4 +27,6 @@ class MockAppConfig(runModeConfiguration: Configuration) extends AppConfig {
   override val registrationUrl: String = "http://localhost:1502/ngr-login-register-frontend/register"
   override val dashboardUrl: String = "http://localhost:1503/ngr-dashboard-frontend/dashboard"
 
+  override val nextGenerationRatesNotifyUrl: String = "http://localhost:1515"
```

Comment:
@david-walters-HMRC 
we do not need MOckConfig, when we inject AppConfig it gives the same values anyways